ios/Podfile

```diff
@@ -0,0 +1,37 @@
+# This was the iOS Deployment Target (project > ZulipMobile > Info in Xcode)
+# as of 2020-03.
+platform :ios, '10.3'
+
+require_relative '../node_modules/react-native-unimodules/cocoapods.rb'
+
+# See
+# https://github.com/facebook/react-native-website/blob/ded79d2cf4456d8b1a4f67c2cdc1391789e70617/docs/integration-with-existing-apps.md#configuring-cocoapods-dependencies
+# for the choice of pods here.
+target 'ZulipMobile' do
+  pod 'React', :path => '../node_modules/react-native', :subspecs => [
+    'Core',
+    'CxxBridge', # Include this for RN >= 0.47
+    'DevSupport', # Include this to enable In-App Devmenu if RN >= 0.43
+    'RCTText',
+    'RCTNetwork',
+    'RCTWebSocket', # Needed for debugging
+    'RCTAnimation', # Needed for FlatList and animations running on native UI thread
+    'RCTImage', # Included in react-native-unimodules' example Podfile
+    # Add any other subspecs you want to use in your project
+  ]
+  # Explicitly include Yoga if you are using RN >= 0.42.0
+  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
```

Comment:
OK. It's very difficult to find the officially recommended example Podfile for RN v0.59 (see https://github.com/facebook/react-native-website/issues/1603#issuecomment-601491956), so including all of its details (with those outdated comments) was a goal, in case we encountered any issues with these dependencies in the future.

But I do link to that officially recommended example Podfile for RN v0.59, in a code comment at the top, which I guess is enough. And it's true that we're >= each of those versions. I would just hope that if, e.g., 'CxxBridge' starts causing an issue, we don't find some SO answer that says we can just remove it, and do that.

Reply:
Yeah. There's that link; and also when asking that kind of "why did we put this here?" question, that's always a good prompt to go looking in the Git history, so that'll give us the full context of why we added these lines.

Also ideally when we've fully digested this migration, each subspec we mention here will be here because we really do need it, and have a clear reason why. :slightly_smiling_face: For a chunk of them the reason might be like "these are core pieces of RN used for everything", but which ancient version they were introduced in isn't really part of that information. In our current state of knowledge, basically that link is the explanation.